doc/classes/Node3D.xml

```diff
@@ -142,91 +145,96 @@
+				Rotates this node's [member basis] around the Z axis by the given [param angle], in radians. This operation is calculated in parent space (relative to the parent) and preserves the [member position].
 			</description>
 		</method>
 		<method name="scale_object_local">
 			<return type="void" />
 			<param index="0" name="scale" type="Vector3" />
 			<description>
-				Scales the local transformation by given 3D scale factors in object-local coordinate system.
+				Scales this node's [member basis] by the given [param scale] factor. This operation is calculated in local space (relative to this node) and preserves the [member position].
 			</description>
 		</method>
 		<method name="set_disable_scale">
 			<return type="void" />
 			<param index="0" name="disable" type="bool" />
 			<description>
-				Sets whether the node uses a scale of [code](1, 1, 1)[/code] or its local transformation scale. Changes to the local transformation scale are preserved.
+				If [code]true[/code], this node's [member global_transform] is automatically orthonormalized. This results in this node not appearing distorted, as if its global scale were set to the default. See also [method is_scale_disabled] and [method orthonormalize].
+				[b]Note:[/b] [member transform] is not affected by this setting.
 			</description>
 		</method>
 		<method name="set_identity">
 			<return type="void" />
 			<description>
-				Reset all transformations for this node (sets its [Transform3D] to the identity matrix).
+				Sets this node's [member transform] to [constant Transform3D.IDENTITY], which resets all transformations in parent space ([member position], [member rotation], and [member scale]).
 			</description>
 		</method>
 		<method name="set_ignore_transform_notification">
 			<return type="void" />
 			<param index="0" name="enabled" type="bool" />
 			<description>
-				Sets whether the node ignores notification that its transformation (global or local) changed.
+				If [code]true[/code], the node will not receive [constant NOTIFICATION_TRANSFORM_CHANGED] or [constant NOTIFICATION_LOCAL_TRANSFORM_CHANGED].
+				It may useful to call this method when handling these notifications to prevent infinite recursion.
 			</description>
 		</method>
 		<method name="set_notify_local_transform">
 			<return type="void" />
 			<param index="0" name="enable" type="bool" />
 			<description>
-				Sets whether the node notifies about its local transformation changes. [Node3D] will not propagate this by default.
+				If [code]true[/code], the node will receive [constant NOTIFICATION_LOCAL_TRANSFORM_CHANGED] whenever [member transform] changes.
+				[b]Note:[/b] Some 3D nodes such as [CSGShape3D] or [CollisionShape3D] automatically enable this to function correctly.
 			</description>
 		</method>
-		<method name="set_notify_transform">
+		<method name="set_notify_transform" keywords="set_notify_global_transform">
```

Comment:
This is out-of-scope for a docs PR but I think we should really rename this to `set_notify_global_transform` and then keep `set_notify_transform` as an alias.

Reply:
Agree. This one is nasty but at least addressable any time unlike the `global_scale` method.